src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -104,9 +104,9 @@
 
   <!-- Common args used for building and testing. -->
   <PropertyGroup>
-    <!-- TODO: Remove the DotNetBuildSourceOnly condition when the default build behavior for source-only builds changes to dev.
-         https://github.com/dotnet/source-build/issues/4855 -->
-    <CommonArgs Condition="'$(ContinuousIntegrationBuild)' == 'true' or '$(DotNetBuildSourceOnly)' == 'true'">$(FlagParameterPrefix)ci</CommonArgs>
+    <!-- TODO: Remove this import and the EnableDevBuildAsDefaultForSourceOnly when dev builds are the default for source only.
+         https://github.com/dotnet/source-build/issues/4922 -->
```

Comment:
nit: the comment refers to an Import but below is a property group. The same is true for the change in L176

Reply:
Fixed with https://github.com/dotnet/sdk/pull/48327/commits/2e4029b6f2e631fa3a1c9d0a9b2c132f917d6f37